src/box/lua/info.c

```diff
@@ -475,7 +475,8 @@ lbox_info_memory_call(struct lua_State *L)
 	lua_settable(L, -3);
 
 	lua_pushstring(L, "lua");
-	lua_pushinteger(L, G(L)->gc.total);
+	lua_pushinteger(
+		L, (lua_getgccount(L) << 10) + lua_gc(L, LUA_GCCOUNTB, 0));
```

Comment:
I would reformat like this:

```diff
--- a/src/box/lua/info.c
+++ b/src/box/lua/info.c
@@ -475,8 +475,8 @@ lbox_info_memory_call(struct lua_State *L)
        lua_settable(L, -3);
 
        lua_pushstring(L, "lua");
-       lua_pushinteger(
-               L, (lua_getgccount(L) << 10) + lua_gc(L, LUA_GCCOUNTB, 0));
+       int countb = (lua_getgccount(L) << 10) + lua_gc(L, LUA_GCCOUNTB, 0);
+       lua_pushinteger(L, countb);
        lua_settable(L, -3);
 
        return 1;
```

the same below

Reply:
Also, I suggest declaring a new function: `luaT_getgccount(L)` in <src/lua/utils.c> and using it. It is also useful for #10223.